syntax = "proto3";

package auth;


// Request Data to send 'Create User' and 'Login'
message RequestDataUser {
  string email = 1;
  string password = 2;
}

// Response after 'Create User'
message ResponseCreateUser {
  int32 status = 1;
  string message = 2;
}

// Response after Login
message ResponseLogin { string access_token = 1; }

// Request 'Create User'
service RequestCreateUser {
  rpc CreateUser (RequestDataUser) returns (ResponseCreateUser) {}
}

// Request 'Login'
service RequestLoginUser {
  rpc LoginUser (RequestDataUser) returns (ResponseLogin) {}
}